cmd/renterd/main.go

```diff
@@ -492,18 +444,18 @@ func main() {
 		logger.Warn("renterd is running on a system without AVX2 support, performance may be degraded")
 	}
 
-	// configure database logger
-	dbLogCfg := cfg.Log.Database
-	if cfg.Database.Log != (config.DatabaseLog{}) {
-		dbLogCfg = cfg.Database.Log
+	if cfg.Log.Database.Level == "" {
```

Comment:
Reminder to manually test the various log level settings we have (regular ones and deprecated) to see if the order they are applied in is still the same as before.

Reply:
I did that. I think it's all good. I'm testing permutations of  `./renterd --openui=false --log.level=debug --db.logger.logLevel=debug --log.database.level=warn`.